sessions/tidymodels/tidyclust-expanding-tidymodels-to-clustering.md

```diff
@@ -39,12 +39,12 @@ Please write abstract below. You may use simple markdown (links, code style, bol
 -->
 
 My name is Emil Hvitfeldt and I have spend the better part of this year working
-with Kelly Bodwin to write the celery package which brings clustering to the
+with Kelly Bodwin to write the tidyclust package which brings clustering to the
 tidymodels framework.
 
 Said framework marked a leap forward for making machine learning methods more
 accessible to a general audiences in R However, it is currently limited to the
 realm of supervised learning.
 
-This talk marks the grand introduction of celery, showing functionality and
+This talk marks the grand introduction of tidyclust, showing functionality and
 workflows that immediately should feel familiar to a tidymodels user
```

Comment:
Hi @EmilHvitfeldt — everything looks great. I took a moment to read through your abstract. What do you think of the following proposal?

> This talk marks the grand introduction of tidyclust, a new package that provides a tidy unified interface to clustering model within the tidymodels framework.
> 
> While tidymodels has been a leap forward in making machine learning methods accessible to a general audience in R, it is currently limited to the realm of supervised learning. tidyclust, by Emil Hvitfeldt and Kelly Bodwin, builds upon the interfaces familiar to tidymodels users to make unsupervised clustering models equally approachable.

Reply:
looks good!